index.js

```diff
@@ -0,0 +1,113 @@
+'use strict'
+const path = require('path')
+const statusboard = require('@pkgjs/statusboard')
+
+module.exports = async (c) => {
+  return statusboard(await config(c))
+}
+
+const config = module.exports.config = async (c = {}) => {
+  const buildDir = path.join(__dirname, 'build')
+
+  return Object.assign({
+    // Status Board Config
+    db: path.join(buildDir, 'data', 'data.db'),
+    baseUrl: '/statusboard',
+    outputDirectory: buildDir,
+
+    // Project configs
+    title: 'Express',
+    description: '',
+
+    issueLabels: ['top priority', 'good first issue', 'help wanted', 'discuss', 'meeting'],
+
+    projects: [
+      'expressjs/express',
+      'expressjs/discussions',
+      'expressjs/cors',
+      'expressjs/morgan',
+      'expressjs/response-time',
+      'expressjs/session',
+      'expressjs/multer',
+      'expressjs/body-parser',
+      'expressjs/compression',
+      'expressjs/serve-static',
+      'expressjs/errorhandler',
+      'expressjs/serve-index',
+      'expressjs/csurf',
+      'expressjs/timeout',
+      'expressjs/flash',
+      'expressjs/vhost',
+
+      'pillarjs/router',
+      'pillarjs/cookies',
+      'pillarjs/send',
+      'pillarjs/finalhandler',
+      'pillarjs/path-to-regexp',
+
+      'jshttp/cookie',
+      'jshttp/fresh',
+      'jshttp/content-disposition',
+      'jshttp/mime-db',
+      'jshttp/media-typer',
+      'jshttp/range-parser',
+      'jshttp/type-is',
+      'jshttp/accepts',
+      'jshttp/negotiator',
+      'jshttp/mime-types',
+      'jshttp/compressible',
+      'jshttp/content-type',
+      'jshttp/http-errors',
+      'jshttp/proxy-addr',
+      'jshttp/etag',
+      'jshttp/forwarded',
+      'jshttp/on-finished',
+      'jshttp/on-headers',
+      'jshttp/vary',
+      'jshttp/basic-auth',
+      'jshttp/methods',
+      'jshttp/statuses'
+    ],
+
+    teams: {
```

Comment:
All of this is also not used currently by the status board.  But my intent was to use this to display an updated list of members.  Now that I look at it, it would be better to have the status board pull the list of teams from GitHub, not the other way around.